### PR TITLE
Config updates

### PR DESCRIPTION
**Pull request by zendoodles**

### CODE REVIEW

File: conf/drupal/config/core.extension.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ module:
   config: 0
   contact: 0
   contact_formatter: 0
-  content_moderation: 0
```
Comment (labbydev) [inline conf/drupal/config/core.extension.yml]: We cannot install without this.

File: conf/drupal/config/core.extension.yml [hunk 2]
```diff
@@ -35,7 +34,6 @@ module:
   mailchimp_campaign: 0
   mailchimp_lists: 0
   mailchimp_signup: 0
-  masquerade: 0
```
Comment (labbydev) [inline conf/drupal/config/core.extension.yml]: I cannot install without this

File: conf/drupal/config/core.extension.yml [hunk 3]
```diff
@@ -58,7 +56,6 @@ module:
   views_ui: 0
   viewsreference: 0
   workbench_tabs: 0
-  workflows: 0
```
Comment (labbydev) [inline conf/drupal/config/core.extension.yml]: I cannot install without this.

**Comment by froboy** [comment]
After merging develop this is now showing only creates and updates on `drush cim`. I've also reversed the module changes. There's still a bunch on prod that's not here, but let's start with this for now.

**Pull request merged**